### PR TITLE
Adding django-pytest

### LINKED ISSUE #20: Adding PyTest
- Adding pytest to update pipfile and pipfile.lock files.
- Adding pytest to run tests automatically for each PR.

### PR DESCRIPTION
**Pull request by rawadabu**
resolve #20 

- Added django-pytest to detect and runs the tests which also provides detailed run report.

### CODE REVIEW

**Review by NitzanMordhai — CHANGES_REQUESTED** [review]
Please change PR description and commit to being more informative, it doesn't say anything about the change.

**Comment by rawadabu** [comment]
> Please change PR description and commit to being more informative, it doesn't say anything about the change.

Commit description edited and squashed the 2 commits.

**Comment by Shalevro2** [comment]
We need to add also the pytest.yml to .github/workflows directory?
You can find the pytest.yml file at the end of DjangoTestingTutoiral.pdf

**Comment by rawadabu** [comment]
> 

Right, created and haven’t added to the branch, thanks for your review @Shalevro2.

File: pytest.ini [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[pytest]
+DJANGO_SETTINGS_MODULE = finalProject_g4.settings
+python_files = tests.py test_*.py *_tests.py
```
Comment (Yarboa) [inline pytest.ini]: Can you please add a file just make sure that github actions will pass success.
Other wise PR will not move to MERGE
Please refer this 
https://github.com/redhat-beyond/Smarticle/blob/main/CONTRIBUTING.md#guidelines-for-quick-pr-merge

Bullet 5

Verify PR checks all passed with success

**Comment by rawadabu** [comment]
So basically PyTest failing for not attaching any test code to the files added to the PR?

**Comment by Anas-mhameed** [comment]
Yes I think , Let's add a simple test like : 
def test_example01():
  assert 1 == 1

**Review by Yarboa — CHANGES_REQUESTED** [review]

File: .github/workflows/pytest.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+name: PyTest
+on: pull_request
+jobs:
+  pytest:
+    name: Run the application's tests with PyTest
```
Comment (Yarboa) [inline .github/workflows/pytest.yml]: Please consider shorter name
such as:
Run PyTest

**Review by Shalevro2 — APPROVED** [review]
LGTM

**Review by Anas-mhameed — APPROVED** [review]
LGTM

**Review by finziamit — APPROVED** [review]
LGTM

**Review by yaelnoyman1 — APPROVED** [review]
LGTM

**Review by Yarboa — APPROVED** [review]

**Comment by Yarboa** [comment]
> Please change PR description and commit to being more informative, it doesn't say anything about the change.

@NitzanMordhai  WDYT? Can you resolve? and merge?

File: pytest.ini (hunk 1 above)
Comment (avi-biton) [inline pytest.ini]: Please add a new line at the end of the file

Comment (rawadabu) [reply pytest.ini]: @avi-biton Done.

Comment (Yarboa) [reply pytest.ini]: LGTM

**Review by NitzanMordhai — APPROVED** [review]

**Pull request merged**